vpr/src/device/rr_graph_view.h

```diff
@@ -123,7 +123,10 @@ class RRGraphView {
     inline short node_yhigh(RRNodeId node) const {
         return node_storage_.node_yhigh(node);
     }
-
+    /** @brief Get the first edge of resource node. This function is inlined for runtime optimization. */
```

Comment:
Can you check if this API gets the first incoming edge or the outcoming edge of a node? I remember it is for outcoming edges. Please confirm. We should this to the comments.

Reply:
It will get the first out coming edge of a node.